github/config.go

```diff
@@ -10,36 +10,47 @@ import (
 )
 
 type Config struct {
-	Token        string
-	Organization string
-	BaseURL      string
+	Token   string
+	Owner   string
+	BaseURL string
 }
 
-type Organization struct {
-	name        string
-	client      *github.Client
-	StopContext context.Context
+type Owner struct {
+	name         string
+	client       *github.Client
+	StopContext  context.Context
+	Organization bool
+}
+
+func (o *Owner) IsOrganization() bool {
+	return o.Organization
```

Comment:
Is there any particular benefit to having this function, when we can just access the exported attribute?